tests/e2e/specs/miscellaneous/WorkspaceWithParent.spec.ts

```diff
@@ -70,30 +66,31 @@ suite('Workspace using a parent test suite', function (): void {
 		const firstExpectedQuickPick: QuickPickItem | undefined = await input.findQuickPick('1. This command from the devfile');
 		const secondExpectedQuickPick: QuickPickItem | undefined = await input.findQuickPick('2. This command from the parent');
 		expect(firstExpectedQuickPick).not.eqls(undefined);
-		expect(secondExpectedQuickPick).not.eqls(undefined);
+		expect(secondExpectedQuickPick).not.undefined;
 	});
 
 	test('Check expected containers in the parent POD', function (): void {
 		const getPodNameCommand: string = `${API_TEST_CONSTANTS.TS_API_TEST_KUBERNETES_COMMAND_LINE_TOOL} get pods --selector=controller.devfile.io/devworkspace_name=sample-using-parent --output jsonpath=\'{.items[0].metadata.name}\'`;
 
-		podName = executeArbitraryShellScript(getPodNameCommand);
-		const containerNames: string = executeArbitraryShellScript(
+		podName = shellExecutor.executeArbitraryShellScript(getPodNameCommand);
+		const containerNames: string = shellExecutor.executeArbitraryShellScript(
 			`${API_TEST_CONSTANTS.TS_API_TEST_KUBERNETES_COMMAND_LINE_TOOL} get pod ${podName} --output jsonpath=\'{.spec.containers[*].name}\'`
 		);
 		expect(containerNames).contain('tools');
 		expect(containerNames).contains('che-gateway');
```

Comment:
> expect(containerNames).contain('tools');
> expect(containerNames).contains('che-gateway');

> expect(envList).contain('DEVFILE_ENV_VAR=true');
> expect(envList).contain('PARENT_ENV_VAR=true');

this can be replaced with .and:
` expect(envList).contain('DEVFILE_ENV_VAR=true').and.contain('PARENT_ENV_VAR=true');`